packages/ocamlformat/ocamlformat.0.28.1/opam

```diff
@@ -25,7 +25,7 @@ license: ["MIT" "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"]
 homepage: "https://github.com/ocaml-ppx/ocamlformat"
 bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
 depends: [
-  "ocaml" {>= "4.08" & < "5.4"}
+  "ocaml" {>= "5.0"}
```

Comment:
@edwintorok this is the artificial limitation to avoid reformatting the code in xapi before we're ready